xpk.py

```diff
@@ -1419,6 +1623,28 @@ def run_gke_node_pool_create_command(args, system) -> int:
     commands.append(command)
     task_names.append(task)
 
+  desired_pw_cpu_node_pools = ['cpu-user-np', 'cpu-rm-np', 'cpu-proxy-np']
```

Comment:
This snippet should "update" an existing cluster to add Pathways CPU nodepools as well, without explicitly deleting and recreating the cluster (as long as tpu-type remains the same). @shaurya89